### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by thclark** [django-guardian/django-guardian#818]
All tests are failing on the `devel` branch. We should address this prior to releasing `2.5.0`.

**Comment by cmaggiulli** [comment]
Hopefully wasnt my MR...  note to any onlookers - Im slow.  So even though I self-assigned this to me, if you eager to fix please do so.  Please comment here so we don't duplicate efforts

**Comment by giancastro** [comment]
Idk if the problem that I'm having is related to that but some bugs are beginning to appear on my app. For instance, I always used the shortcut get_perms, but now it's returning all possible permissions instead of the current ones. The weird thing is that the get_user_perms shortcut is working as expected.

```python
In [49]: shortcuts.get_perms(user, easement)
Out[49]: ['add_easement', 'change_easement', 'delete_easement', 'view_easement']

In [50]: shortcuts.get_user_perms(user, easement)
Out[50]: <QuerySet ['view_easement']>
```

The get_obj_perms tag is also having the same bug.

**Comment by BonaFideIT** [comment]
Fixed in #827, can be closed after merging.
